src/utils/chatbot.ts

```diff
@@ -0,0 +1,30 @@
+import { ChatBotMessage, ChatbotRole } from '@graasp/sdk';
+
+import { APP_DATA_TYPES } from '../config/appDataTypes';
+import { ThreadMessage } from '../interfaces/chatbot';
+
+// TODO: maybe it is possible to move to SDK and to reuse it in code-capsule
```

Comment:
Yes that would be nice ! (Can be done in another PR)

Reply:
I opened an issue in the apps-query-client and I will migrate in another PR.